src/emc/usr_intf/axis/scripts/axis.py

```diff
@@ -2931,19 +2933,13 @@ def set_rapidrate(n):
 
 def activate_ja_widget_or_set_feedrate(jora):
     # note: call with integers only
-    if manual_ok():
-        if joints_mode():
-            activate_ja_widget(jora)
-            return
-        if not joints_mode():
-            if s.kinematics_type == linuxcnc.KINEMATICS_IDENTITY:
-                activate_ja_widget(jora)
-                return
-            else:
-                return # ignore number (no teleop letter correspondence)
+    if joints_mode() and s.kinematics_type != linuxcnc.KINEMATICS_IDENTITY:
+        if jora == 0: return
```

Comment:
This line breaks using "`" to select joint number zero, which current users will be accustomed to.  Remove it.